modules/minion/tests/minion/TaskTest.php

```diff
@@ -67,4 +109,38 @@ public function test_convert_class_to_task($expected, $class)
 	{
 		$this->assertSame($expected, Minion_Task::convert_class_to_task($class));
 	}
+	
+	/**
+	 * Provides test data for test_set_domain_name()
+	 *
+	 * @return array
+	 */
+	public function provider_set_domain_name()
+	{
+		return [
+			['https://www.example.com/welcome', 'https://www.example.com', 'welcome'],
+			['http://www.example.com/welcome', 'http://www.example.com', 'welcome'],
+			['http://www.example2.com/welcome', NULL, 'welcome'],
+			['http://www.example.com:8080/welcome', 'http://www.example.com:8080', 'welcome'],
+		];
+	}
+
+	/**
+	 * Tests that a task can be converted to a class name
+	 *
+	 * @test
```

Comment:
Remove this, not needed cause function name starts with "test"

Reply:
Damn, missed this one